package/templatectl/package

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+# Copyright (c) 2021 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(templatectl)
+pkgdesc="Tool to add/remove templates for xochitl"
+url=https://github.com/PeterGrace/templatectl
+pkgver=0.1.2-3
+timestamp=2021-01-15T01:23Z
+section="templates"
+maintainer="Peter Grace <pete.grace@gmail.com>"
+license=MIT
+
+image=rust:v1.2.2
+source=(
+    "https://github.com/PeterGrace/templatectl/archive/v${pkgver%-*}.zip"
+    usr-share-remarkable-templates.mount
+)
+sha256sums=(
+    305e3dbf29aea7f762fba22ceebfa004a5f666ae96fdb8776b2acfdb39539469
+    SKIP
+)
+
+build() {
+    cargo build --release
+}
+
+package() {
+    install -D -m 755 -t "$pkgdir"/opt/bin \
+        "$srcdir"/target/armv7-unknown-linux-gnueabihf/release/templatectl
+    install -D -t "$pkgdir"/lib/systemd/system "$srcdir"/usr-share-remarkable-templates.mount
+}
+
+preinstall() {
+    local target_path=share/remarkable/templates
+    local factory_path="/usr/$target_path"
+    if [ ! -d /home/root/.entware/"$target_path" ]; then
+        mkdir -p /home/root/.entware/"$target_path"
+        cp -r /"$factory_path"/* /home/root/.entware/"$target_path"
+    fi
+}
+
+postupgrade() {
+    systemctl stop usr-share-remarkable-templates.mount
```

Comment:
Why do we need this?

Reply:
Because on an upgrade the file may have changed, so we stop it in postupgrade, daemon-reload and then start in configure.

Reply:
Fair enough. This should also go in #243 since I don’t think we’re doing this for all packages that contain systemd services.